### ISSUE TITLE
Submitting ≥11 entries from my «Browse Websites Without Logging In» list

### ISSUE THREAD

**Issue by DandelionSprout** [uBlockOrigin/uAssets#5911]
Having now amassed a few more entries in that list of mine since last time (#5353), I now felt it was time to submit them to uAssets Annoyances again:

```
From «Browse Websites Without Logging In»:

// Fixed
! Twitter
! Removes the "Aren't you on Twitter??" prompt in the upper right of user timelines
! https://twitter.com/nasuno_Posi
twitter.com##.js-signup-call-out
// Fixed

// Fixed
! Twitter Mobile
! Removes the "Are you new on Twitter?" banner in the upper right of user timelines (Screen area has to be ≥~1400px wide)
! https://mobile.twitter.com/nasuno_Posi
mobile.twitter.com##div[data-testid="sidebarColumn"] section[aria-label][role="region"]
// Fixed

! Tumblr
// Declined
! https://www.tumblr.com/tagged/picrew
! Removes a login prompt in the lower right
tumblr.com##.register_prompt
// Declined
// Fixed
! Makes the bottom image row of tag-searches not cut off when not logged in
tumblr.com##.posts:style(margin-bottom: 0px !important)
// Fixed

// Fixed
! The Australian Frequent Flyer
! https://www.australianfrequentflyer.com.au/community/whats-new/latest-activity
! Removes a login info banner on the top of some pages
australianfrequentflyer.com.au##.focus-guest-alert.focus-guest
// Fixed

// Fixed
! FM.to
! https://github.com/DandelionSprout/adfilt/issues/7#issuecomment-483040002
! Removes an uncloseable login nag that occurs after 30sec
fm.to###popup_guest
fm.to##.modal-backdrop
// Fixed

// Fixed
! MySQL
! https://dev.mysql.com/downloads/file/?id=486042
dev.mysql.com##.login-callout
// Fixed

// Fixed
! YouTube
! https://new.reddit.com/r/uBlockOrigin/comments/bet7iu/
! Removes a short-lasting note about being logged off
youtube.com##ytd-popup-container > .ytd-popup-container > #contentWrapper > .ytd-popup-container[position-type="OPEN_POPUP_POSITION_BOTTOMLEFT"]
! https://github.com/DandelionSprout/adfilt/issues/32#issuecomment-482908407
! Removes a relatively large login nag module
m.youtube.com###mealbar\:3 > ytm-mealbar.mealbar-promo-renderer
// Fixed

// Fixed in Adguard Annoyances
! Reverso Context
! https://context.reverso.net/translation/english-french/bread
! Removes a small login prompt in the bottom left
context.reverso.net###register-popup-bottom
// Fixed in Adguard Annoyances
```

I now hope that I've added sufficient contexts and example links, that none of them depend on non-uBO extensions, and that none of the example links are NSFW; so that there hopefully won't be any problems with reproducing the entries.

While I've only tested a few of these entries while having AdGuard's and Fanboy's annoyances lists turned on, I get the perception that these entries aren't the kind of stuff that those lists tend to deal with.

### Screenshot(s)

Can be added for specific sites/entries on request.

### Versions

- Browser/version: Chrome 75.0.3770.100 64-bit stable for Windows
- uBlock Origin version: Nano Adblocker 1.0.0.118 + Nano Defender 15.0.0.153

### Settings

Lists: 
![image](https://user-images.githubusercontent.com/22780683/60598404-f4e38d00-9dac-11e9-92fc-f7d3a500ec98.png)
Advanced settings: 
![image](https://user-images.githubusercontent.com/22780683/60598453-09c02080-9dad-11e9-9140-2cad868409f2.png)

### Notes

Just for the record, I have had problems trying to get `:has` entries for Twitter Mobile to apply on page reloads, and with getting hiding entries for Twitter Mobile to show up in the logger (Screenshot of the latter problem below; log entries in the screencap deviate slightly from those of the list setup above). I have not personally been able to identify the causes of either issue in a hurry, but I wouldn't be surprised if the ungodly long element names that Twitter Mobile has could be a factor in it. 
![image](https://user-images.githubusercontent.com/22780683/60599145-388ac680-9dae-11e9-895d-af0b82439374.png)

**Comment by okiehsch** [comment]
> ! Tumblr
! https://www.tumblr.com/tagged/picrew
! Removes a login prompt in the lower right

I would consider that prompt informational, it explains why the user can't scroll down.
Your style filter can be added.

> ! Reverso Context
! https://context.reverso.net/translation/english-french/bread
! Removes a small login prompt in the bottom left

Already fixed in `Adguard Annoyances`.

> ! FM.to
! https://github.com/DandelionSprout/adfilt/issues/7#issuecomment-483040002
! Removes an uncloseable login nag that occurs after 30sec

`fm.to##+js(setTimeout-defuser.js, popup_guest, 30000)`
prevents the login nag from appearing in the first place and is the preferable fix n my opinion.

**Comment by krystian3w** [comment]
PS. old filter for YouTube cut pop-up with new "terms of use" for logged users.

```
youtube.com##paper-dialog:has(ytd-mealbar-promo-renderer)
```

known as amethyst / purple / violet 🐙.

**Comment by okiehsch** [comment]
Thanks.

**Comment by krystian3w** [comment]
```
context.reverso.net##.blocked:style(filter: none !important;)
context.reverso.net###blocked-results-banner
```

I cannot found this in AdGuard / Fanboy Annoyance.

![obraz](https://user-images.githubusercontent.com/35370833/61252358-91b70a80-a75d-11e9-9269-4a4836ee1cad.png) | ![Screenshot_2019-07-16 bread - Translation into French - examples English Reverso Context](https://user-images.githubusercontent.com/35370833/61252413-bdd28b80-a75d-11e9-8beb-5ecb3e33f6ee.png)
---- | ---
blured | cleaned (links works, select text works)

**Comment by DandelionSprout** [comment]
Ah, thanks for noticing it, @krystian3w. During the entry roundup before I wrote the OP, I had mistook those entries for paywall bypassers and thought they couldn't be added to uAssets.

**Comment by krystian3w** [comment]
these still are needed:

```adb
tumblr.com##.posts:style(margin-bottom: 0px !important)
```

?

> https://github.com/uBlockOrigin/uAssets/issues/19304



due possible glitch on Firefox: #19304 and nice will be skip report at https://bugzil.la to fix in 115.1.1 ESR and 116.0.3.
